gcip/core/need.py

```diff
@@ -24,8 +24,8 @@
 
 That also mean that stages are ignored, as the `example` stage for example.
 
-However you can use the `Need` class directly when depending on other pipelines jobs or you don't want to [download
-artifacts](https://docs.gitlab.com/ee/ci/yaml/#artifact-downloads-with-needs) from preceding jobs:
+However you have to use the `Need` directly class when depending on other pipelines jobs or for further configuration of the need,
```

Comment:
There is an Error:
`directly class` -> `class directly`

Reply:
```suggestion
However you have to use the `Need` class directly when depending on other pipelines jobs or for further configuration of the need,
```